frontend/src/services/fetchData.ts

```diff
@@ -66,3 +69,39 @@ export const fetchApplicationRoles = async (
     ).data;
     return applicationRoles;
 };
+
+export const fetchApplicationAdmins = async (
+    applicationId: number | undefined
```

Comment:
Probably just `applicationId: number` should be fine, no need for `undefined` I think.

Reply:
Thanks for the comment, Ian! Just removed any unnecessary code related.